content/blog/cdf-intro-uber-crane/index.md

```diff
@@ -55,19 +54,20 @@ Last but not least, our SaaS product FABR Infra offers great convenience for dev
 
 ## Example Setup with FABR
 
-Given Uber didn't share specifics about their architecture we'll make something up based what they did not share. 
+Given that the post did not share specifics about Uber's architecture, let's use a very simple infrastructure setup here for ease of understanding:
 
-- A public-facing web server on cloud vendor A.
-- An on-prem database and another database on cloud vendor B;
-- The web server has connectivity to both databases.
+- A public-facing web server on cloud vendor A;
+- An on-prem database;
+- Another database on cloud vendor B;
+- The web server should have connectivity to both databases.
 
-Remember the requirements repeatable infrastructure, layered dependencies, config, and change management. Also, we picked CDK TF and Golang.
+Given the on-prem and multi-cloud nature of the infrastructure mentioned in the post, we will pick Terraform CDK as the imperative IaC tool, with some help from Golang which is also mentioned in the post.
```

Comment:
Changing the tone a little, to be more driven by us instead of following and second-guessing about Uber's.

Reply:
I don't mind the edited version, but don't think the first is any less driven or second guessing. It's simply explicitly stating a summary of the requirements Uber build Crane against though is missing "multi-cloud and on-prem".